Create ROMappingPost repository, service, controller test code

situodys opened the pull request:
ROMappingPost repository, serivce, controller test code

situodys committed:
Test: ROMappingPost repository test

- post 검색 및 페이징 처리, 현재는 cafe 정보만 검색 조건 설정

situodys committed:
Test: ROMappingPost service test

- 각 entity 내부 toDTO 구현

situodys committed:
Test: ROMappingPost controller test

Tianea2160 commented on src/main/java/com/sanhak/backend/domain/post/controller/ROMappingPostController.java:
```diff
@@ -18,15 +22,18 @@ public class ROMappingPostController {
+        Page<PostDTO> roMappingPosts = roMappingPostService.searchPostWithPagination(postSearch);
+        PostResponseDTO response = new PostResponseDTO(roMappingPosts);
+
+        return new ResponseEntity<>(response, HttpStatus.OK);
```
아래에는 ResponseEntity.ok(postDTO)로 적어 놓으셨는데 위에는 다르게 적으셔서 쓰는 방식을 하나로 통일하는 것도 좋을 거 같습니다.

situodys replied:
29라인같은 형태로 통일하도록 할게요

Tianea2160 commented on src/main/java/com/sanhak/backend/domain/post/dto/PostResponseDTO.java:
```diff
@@ -0,0 +1,51 @@
+
+@Data
+@NoArgsConstructor
+public class PostResponseDTO{
```
PostResDTO라고 네이밍을 조금 간단히 줄여도 좋을거 같아요 DTO가 이름이 너무 길어지는 거 같아서 제가 다른 DTO들도 최대한 이름을 줄이고 있어요.
Response -> Res, Create -> Crt, Update ->Upd이런식으로 3글자 정도로 줄이는 컨벤션 어떠신가요?

situodys replied:
네넵 수정할게요

situodys committed:
Fix: modify review request

- entity.toDto() 대신 modelMapper로 dto 변환
- ResponseEntity 생성 통일
- Service에서 entity -> dto 변환

situodys committed:
Refactor: name PostResponseDTO->PostResDTO

Tianea2160 committed:
Merge branch 'develop' into feature-post-test
